packages/documentation/docs/controls/_textarea-field_styleguide.mdx

```diff
@@ -0,0 +1,62 @@
+The Textarea component allows users to input multi-line text. It can be used in various forms where longer text input is required.
+
+![Textarea overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3814-1128&t=DtCmoFcLwhf7ke3S-4)
+1. Label
+2. Required field indicator (optional)
+3. Placeholder text
+4. Container
+5. Resize handle
+6. Helper or feedback text
+7. Counter
+
+## Options
+
+- **Label**: [See form field](./forms/forms-field.md)
+- **Value**: [See form field](./forms/forms-field.md)
+- **Required**: [See form field](./forms/forms-field.md)
+- **Helper text**: [See form field](./forms/forms-field.md)
+- **Feedback text**: [See form field](./forms/forms-field.md)
+- **Show text as tooltip**: [See form field](./forms/forms-field.md)
+- **Placeholder text**: [See form field](./forms/forms-field.md)
+- **Counter**: [See form field](./forms/forms-field.md)
+- **Resize behavior**: Determines how textareas can be resized (both directions, horizontally, vertically, no resizing)
+- **Columns and width**: Defines inital width by number of columns and/or width
+- **Rows and height**: Defines inital height by number of rows and/or height
+
+
+## Behavior
+
+- **Interaction**:
+  - Clicking in the container enables the editing of the field
+  - Users can type, copy, paste, and cut text within textareas
+  - Optional: Users can resize textareas to fit their needs
+- **Validation**:
+  - Minimum and maximum length defines number of characters allowed
+  - [See form validation](./forms/forms-validation.md)
+- **Overflow**: Text within the textarea is not truncated; it supports scrolling for overflow content
+- **Alignment**: Left aligned
```

Comment:
Can the alignment be adjusted

Reply:
In general, please use periods. Complete sentences should always be used in every section, except in the ‘do and don’t’ section.